Podfile

```diff
@@ -1,9 +1,10 @@
-platform :ios, '8.0'
+platform :ios, '9.0'
 use_frameworks!
 
 def shared_pods
   #pod 'Mapbox-iOS-SDK', '~> 3.6.3'
   pod 'Mapbox-iOS-SDK-symbols', :podspec => 'https://raw.githubusercontent.com/mapbox/mapbox-gl-native/ios-v3.6.4/platform/ios/Mapbox-iOS-SDK-symbols.podspec'
+  pod 'MapboxNavigation', '~> 0.10'
```

Comment:
This is apparently problematic — tests are failing because CocoaPods is trying to install Mapbox.framework twice.

Reply:
`shared_pods` is installed across all targets, while `MapboxNavigation` is only needed for DocsCode.